### PR TITLE
Fix a typo

### PR DESCRIPTION
**Pull request by lu-zero**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -240,7 +240,7 @@ fn install(args: InstallOpts) -> Result<()> {
         bail!("{} Failed to save config {:#}", emoji::ERROR, e);
     }
 
-    info!("{} Installation suscesfully completed!", emoji::CHECK);
+    info!("{} Installation succesfully completed!", emoji::CHECK);
```
Comment (SergioGasquez) [inline src/main.rs:243]:
```suggestion
    info!("{} Installation successfully completed!", emoji::CHECK);
```

File: src/main.rs [hunk 2]
```diff
@@ -298,7 +298,7 @@ fn uninstall(args: UninstallOpts) -> Result<()> {
     let conf_file = conf_dirs.config_dir().join("espup.toml");
     remove_file(conf_file)?;
 
-    info!("{} Uninstallation suscesfully completed!", emoji::CHECK);
+    info!("{} Uninstallation succesfully completed!", emoji::CHECK);
```
Comment (SergioGasquez) [inline src/main.rs:301]:
```suggestion
    info!("{} Uninstallation successfully completed!", emoji::CHECK);
```

File: src/main.rs [hunk 3]
```diff
@@ -344,7 +344,7 @@ fn update(args: UpdateOpts) -> Result<()> {
         bail!("{} Failed to save config {:#}", emoji::ERROR, e);
     }
 
-    info!("{} Update suscesfully completed!", emoji::CHECK);
+    info!("{} Update succesfully completed!", emoji::CHECK);
```
Comment (SergioGasquez) [inline src/main.rs:347]:
```suggestion
    info!("{} Update successfully completed!", emoji::CHECK);
```

**Review by SergioGasquez — CHANGES_REQUESTED** [review]
Hi, good catch, and thanks for your contribution! Do you mind addressing the suggestions?

**Comment by lu-zero** [comment]
Sure, I managed to put a typo in the sed expression ^^;

**Review by SergioGasquez — APPROVED** [review]

**Pull request merged**